leakcanary-android/src/main/java/com/squareup/leakcanary/AndroidWatchExecutor.java

```diff
@@ -21,24 +21,32 @@
 import android.os.MessageQueue;
 import java.util.concurrent.Executor;
 
+import static java.util.concurrent.TimeUnit.SECONDS;
+
 /**
  * {@link Executor} suitable for watching Android reference leaks. This executor waits for the main
- * thread to be idle then posts to a serial background thread with a delay of {@link
- * #DELAY_MILLIS} milliseconds.
+ * thread to be idle then posts to a serial background thread with a delay of
+ * {@link R.integer.leak_canary_watch_delay_seconds} seconds.
  */
 public final class AndroidWatchExecutor implements Executor {
 
   static final String LEAK_CANARY_THREAD_NAME = "LeakCanary-Heap-Dump";
-  private static final int DELAY_MILLIS = 5000;
+  private static final int DEFAULT_DELAY_SECONDS = 5;
 
   private final Handler mainHandler;
   private final Handler backgroundHandler;
+  private final long delayMillis;
 
   public AndroidWatchExecutor() {
```

Comment:
Remove this constructor as well as `DEFAULT_DELAY_SECONDS`, no need for a default since we don't even use it. Anyone wanting to customize the executor can pass in their own value.